README.md

```diff
@@ -254,9 +254,13 @@ Python:
 instrumentation.opentelemetry.io/inject-python: "true"
 ```
 
-DotNet:
+.NET:
+.NET auto-instrumentation also honors an annotation that will be used to set the .NET runtime version. There are two versions supported `linux-glibc` and `linux-musl`.
```

Comment:
I know that we think about the standard lib that we are linking against, but, for a .NET user I think it makes more sense to use the [Runtime Identifiers](https://learn.microsoft.com/en-us/dotnet/core/rid-catalog). Two reasons: first it is future proof, second it is the term and values used in .NET itself.
```suggestion
.NET auto-instrumentation also honors an annotation that will be used to set the .NET [Runtime Identifiers](https://learn.microsoft.com/en-us/dotnet/core/rid-catalog)(RIDs). Currently, only two RIDs are supported: `linux-x64` and `linux-musl-x64`.
```

Reply:
176c908030b328b8025ce9afb1b1027bb39c71e0